jdaviz/core/helpers.py

```diff
@@ -35,3 +35,49 @@ def __init__(self, app=None):
 
     def load_data(self, data, parser_reference=None, **kwargs):
         self.app.load_data(data, parser_reference=parser_reference, **kwargs)
+
+    @property
+    def fitted_models(self):
+        """
+        Returns the fitted model parameters.
+
+        Returns
+        -------
+        parameters : dict
+            dict of Quantity arrays, or None.
+        """
+        all_models = {}
+        if hasattr(self.app, '_fitted_1d_models'):
+            all_models = {**all_models, **self.app._fitted_1d_models}
+        if hasattr(self.app, '_fitted_3d_model'):
```

Comment:
Not sure if priority is an issue, but second `if` shouldn't run blindly.
```suggestion
        elif hasattr(self.app, '_fitted_3d_model'):
```